src/components/ui-shell/ui-shell.hbs

```diff
@@ -0,0 +1 @@
+{{> platform-header }}
```

Comment:
Nitpick - Do you want to remove the whitespace right before `}`?

Reply:
Just was following the convention here: https://handlebarsjs.com/partials.html

```hbs
{{> myPartial }}
```